dataplicity/constants.py

```diff
@@ -16,3 +34,18 @@
 # TODO: Replace this with a sensible chunk size once we identify the
 # issue with ssh over Porthole
 CHUNK_SIZE = 1024 * 1024
+
+# Maximum number of services (port forward/commands/file etc)
+LIMIT_SERVICES = get_environ_int("DATAPLICITY_LIMIT_SERVICES", 500)
+
+# Maximum number of terminals (separate pool from services)
+LIMIT_TERMINALS = get_environ_int("DATAPLICITY_LIMIT_TERMINALS", 100)
```

Comment:
Separate pool for terminals. 100 seems reasonable.

Router will kick any terminals with no data transfer after a period of time. So its unlikely to reach this limit.